src/components/common/Swap/Swap.tsx

```diff
@@ -345,7 +345,7 @@ const Swap = () => {
     refetchBalances
   ]);
 
-  const insufficientSellBalance = parseFloat(sellValue) > sellBalanceValue;
+  const insufficientSellBalance = sellBalanceValue.lt(parseFloat(sellValue));
```

Comment:
So `sellBalanceValue` is a decimal now and `sellValue` is float? Should it be like this?

Reply:
Fixed